docs/application/resource.md

```diff
@@ -170,6 +192,16 @@ View the component list in the resource detail page:
 
 For more details on dependency graphs, please refer to [Dependency Graph](/application/graph).
 
-![resource-component-graph](/img/v0.5.0/application/resource/resource-component-graph.png)
+![resource-component-graph](/img/v0.6.0/application/resource/resource-component-graph.png)
+
+![resource-component-graph2](/img/v0.6.0/application/resource/resource-component-graph2.png)
+
+## Resource Runs
+
+Every operation such as deployment, rollback, start, and stop of a resource will generate a run record. You can view all run records and their details on the resource details page.
+
+![resource-run-list](/img/v0.6.0/application/resource/resource-run-list.png)
+
+The resource run includes information such as the status of the run, start time, duration, type of operation, operator, component changes in resources, resource configuration, logs, change history, etc. You can view the detailed information of the run by clicking the operation button.
```

Comment:
The resource run includes information such as the run's status, start time, duration, operation type, operator, component changes in resources, resource configuration, logs, change history, and more. You can view detailed information about the run by clicking the operation button.